hack/lib/constants.sh

```diff
@@ -76,7 +77,6 @@ readonly OPENSHIFT_BINARY_SYMLINKS=(
   osc
   oadm
   osadm
-  kubectl
```

Comment:
what does OC_BINARY_COPY do? looks like it already symlinks oc to kubectl

Reply:
> what does OC_BINARY_COPY do? looks like it already symlinks oc to kubectl

I wasn't going to remove that one yet.  This is practice for finding places to adjust as I remove `kube-apiserver`, `kubelet`, etc in bulk.